jupyter_book/book_template/_includes/buttons/download.html

```diff
@@ -0,0 +1,4 @@
+{% if site.use_download_button -%}
+<a href="{{ page.interact_link | relative_url }}" download>
```

Comment:
@TomDonoghue apparently the `download>` part here is what causes the browser to auto-download instead of trying to open the file

Reply:
That's cool - nice find!